rllib/agents/ppo/ppo_tf_policy.py

```diff
@@ -344,6 +361,15 @@ def setup_mixins(policy: Policy, obs_space: gym.spaces.Space,
     LearningRateSchedule.__init__(policy, config["lr"], config["lr_schedule"])
 
 
+def view_requirements_fn(policy):
```

Comment:
PPO and others need a complete input dict "view" for calculating GAE at the end of the trajectory. This view requirement sets this up.

Reply:
(see `postprocess_trajectory` above)